### PR TITLE
WIP: Minimize !@#%

### PR DESCRIPTION
**Pull request by romain-dartigues**
Minimize some more (not to be merged as-it).

I started hunting bytes, and I'd like a feedback first.
Am I doing right? Still want one MR per icon or could I propose a batch (with one icon per commit)?

* Lucky: remove unused objects, use a gradient instead of three shapes; not supported by svg2android
* XMPP: optimize path, use cloning and transform, unsupported by svg2android
* Intel: optimize path and remove the ®
* Ubiquiti: merge path, remove unused objects

### CODE REVIEW

File: images/svg/xmpp.svg [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg"
+xmlns:x="http://www.w3.org/1999/xlink"
```
Comment (edent) [inline images/svg/xmpp.svg]: This needs to be `xmlns:xlink=...` otherwise some rendering engines get confused.

**Comment by edent** [comment]
It's addictive, isn't it :-)

Happy for you to do them in batches as you seem to know what you're doing.  I've left a note about the XLINK issue. Other than that, they look ready to go.

File: images/svg/xmpp.svg (hunk 1 above)
Comment (romain-dartigues) [reply images/svg/xmpp.svg]: Oh, I took it from Rust (yeah, I know, #348), but ok.

**Comment by edent** [comment]
These all look good. Merge whenever you think they're ready.

**Comment by edent** [comment]
Happy if you want to merge these.

**Comment by edent** [comment]
As it has been more than a year, I'm going to cherry pick some of the logos from here which still need doing.
Sorry for the delay. It has been a strange year...!

**Pull request closed without merging**